addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -87,8 +93,9 @@
      *
      * @param thing Thing object representing a FRITZ! device
      */
-    public DeviceHandler(@NonNull Thing thing) {
+    public DeviceHandler(@NonNull Thing thing, @NonNull HttpClient httpClient) {
```

Comment:
Only `@NonNullByDefault` and `@Nullable` should be used when using null annotations (see [code style **12.**](https://www.eclipse.org/smarthome/documentation/development/guidelines.html#a-code-style)).